### ISSUE TITLE
Get link to build -> 414 Request-URI Too Large

### ISSUE THREAD

**Issue by Sneed69** [iconmaster5326/Chronomancer#15]
Using the link doesn't work

**Comment by iconmaster5326** [comment]
You're right. Some build codes are too large to fit in a standard URL. In the meantime, you can copy the build code from the URL and paste it into the "Import Build" dialog.
